app/src/main/res/values/preference_keys.xml

```diff
@@ -86,4 +86,7 @@
     <string name="pref_key_light_theme" translatable="false">pref_key_light_theme</string>
     <string name="pref_key_dark_theme" translatable="false">pref_key_dark_theme</string>
     <string name="pref_key_default_theme" translatable="false">pref_key_default_theme</string>
+
+    <!--Opened tabs counter used to trigger erase tabs CFR-->
```

Comment:
nit: For consistency with the others this comment should be padded with a space character (between it and the comment tags)